practicalai/practical-ai-74.md

```diff
@@ -92,13 +92,13 @@ One of the things I'd like to know is given that kind of larger team that we've
 
 As I mentioned before, testing the cost of your features also is something that is very important, and aligns very well with these GDPR, also ensuring that your system maintains privacy across the entire pipeline. Sometimes we are very concerned about the privacy of our raw data, because that's our most valuable asset, but you need to ensure that your system or every transformation or every data manipulation that you're doing complies with that privacy as well.
 
-Also, make sure that you are aware of how much time it's taking you develop a new feature or a new production model, because that's also gonna help a lot with \[unintelligible 00:20:59.04\] which also will prevent you from going into half-baked features, or having very tight data jungles as well.
+Also, make sure that you are aware of how much time it's taking you to develop a new feature or a new production model, because that's also gonna help a lot with \[unintelligible 00:20:02.04\], which also will prevent you from going into half-baked features, or having very tight data jungles as well.
 
 If you are, for example, the machine learning engineer, you're gonna be testing for model development practices, and monitoring those models. So then making sure that everything is checked into our repository, that there is version control, making sure that there is a peer-review process... It not only has to be the senior data scientist, but everyone in the team has to be responsible for that, making sure (again) that you're checking your impact metrics, checking the impact of your tunable hyperparameters... And also check against simpler models.
 
-Sometimes, because we are so into deep learning, and deep learning is the most popular framework or the most popular approach at the moment, we want to use very sophisticated models... Sometimes it's also good to go back to the basics and compare against a much more simple, or a simpler model, just to have a baseline and ensure that we are actually going down the right path, and that it makes sense, the additional cost that we're going into \[unintelligible 00:22:19.21\] And again, test for your implicit bias.
+Sometimes, because we are so into deep learning, and deep learning is the most popular framework or the most popular approach at the moment, we want to use very sophisticated models... Sometimes it's also good to go back to the basics and compare against a much more simple, or a simpler model, just to have a baseline and ensure that we are actually going down the right path, and that it makes sense, the additional cost that are going into computage. And again, test for your implicit bias.
```

Comment:
```suggestion
Sometimes, because we are so into deep learning, and deep learning is the most popular framework or the most popular approach at the moment, we want to use very sophisticated models... Sometimes it's also good to go back to the basics and compare against a much more simple, or a simpler model, just to have a baseline and ensure that we are actually going down the right path, and that it makes sense, the additional cost that are going into compute. And again, test for your implicit bias.
```